docs/scale_run_script.md

```diff
@@ -0,0 +1,71 @@
+# Using the scale run script
+
+The [utils/scale_run.sh](../utils/scale_run.sh) script is used to automate the
+execution of scale testing for the various EC2 instance types.  This script
+runs 5 rounds each of cold start and warm start scenarios.  It creates a
+directory for each run.
+
+Out of the box, it will run the following:
+
+* 1 run of 100 instances on a c5.large instance for the trial license
+  (10 hosts) scenario.  Tuning is set to true, but is not forced, so it will
+  not be applied to this instance due to insufficient ram.
+* 5 cold start runs (provisioning on the first) with tune set to true on the
+  following:
+    * c5.xlarge: Starting with 2000 instances
+    * c5.2xlarge: Starting with 4100 instances
+    * c5.4xlarge: Starting with 4100 instances
+* 5 warm start runs (using the provisioned hosts from the cold run above) with
+  tune set to true on the following:
+    * c5.xlarge: Starting with 1800 instances
+    * c5.2xlarge: Starting with 5000 instances
+    * c5.4xlarge: Starting with 6000 instances
+
+This behaviour is executed via the following command:
+```
+./util/scale_run.sh 2019.1.0
+```
+
+
+## Environment Preparation
+
+This script should be executed on a test runner set up as per the
+[gplt setup instructions](slv_environment_setup.md) with adequate
+storage space for all of the logs (100GB recommended).
+
+
+## Command line flags
+
+### No-op mode
+
+The script provides a simple no-op mode that will echo out the ENV variable
+settings as well as the scale command to be executed for each run.  This
+mode is invoked with the `-n` or `--noop` flag.
+
+
+### Tuning
+
+The application of pe-tune can be toggled using the `--tune` or `--notune`
+flags.  Having tuning applied is the default.
+
+
+### Run ID
+
+The script provides the `-i` or `--run-id` flag to enable the user to supply
+an identifier to be used in the naming of the created run directories.  This
+allows the user to run the script multiple times and label the runs in a
+meaningful way for future reference.  For example, a JIRA ticket number might
+be used with this flag.
+
+```
+./util/scale_run.sh -n -i foo 2019.1.0
```

Comment:
you mention above that a jira ticket might be used... but then the example doesn't...

Reply:
doc updated to use JIRA ticket number